draft-rescorla-tls-esni.md

```diff
@@ -250,10 +250,12 @@ example.com, the ESNI TXT Resource Record might be:
 _esni.example.com. 60S IN TXT "..." "..."
 ~~~
 
-Servers SHOULD configure DNS such that, upon querying a domain name
-with ESNI support, at most one each of A, AAAA, TXT ESNI, and 
-ALTSVC {{?I-D.schwartz-httpbis-dns-alt-svc}} Resource Record is 
-returned. Alt-Svc records may be used to inform the client of the 
+Servers MUST ensure that, if multiple A, AAAA or
+ALTSVC {{?I-D.schwartz-httpbis-dns-alt-svc}} records are returned for a domain
```

Comment:
The ALTSVC records don't matter as much here, and probably shouldn't be bound to the MUST. The rest LGTM.